resources/js/widgets/DeathCounter.vue

```diff
@@ -6,7 +6,7 @@
           <li class="list-group-item px-0 border-0">
             <img
               :src="animal.icon"
-              :alt="animal.name + ' icon'"
+              :alt="animal.name"
```

Comment:
It's worth noting that this will not pluralize the animal name, which may have been the intention of the bullet point in the Trello ticket. Some of the animal names are already plural (ex. "Cattle"), I'm unsure if the inconsistency is going to be a problem.